.github/CODEOWNERS

```diff
@@ -245,3 +245,9 @@
 
 # ServiceLabel: %tools-ConfidentialLedger
 # ServiceOwners:                       @taicchoumsft @ivarprudnikov
+
+# PRLabel: %tools-ResourceHealth
+/tools/Azure.Mcp.Tools.ResourceHealth/      @microsoft/azure-mcp
```

Comment:
do a `git blame` and see who added this directory.  They should be the owner.  This entry needs at least 1 real person and not just the team alias